src/log-store/src/kafka/log_store.rs

```diff
@@ -252,3 +284,157 @@ impl LogStore for KafkaLogStore {
         Ok(())
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use common_test_util::get_broker_endpoints;
+    use common_test_util::wal::kafka::topic_decorator::{Affix, TopicDecorator};
+    use common_test_util::wal::kafka::{create_topics, BROKER_ENDPOINTS_KEY};
+    use rand::seq::IteratorRandom;
+
+    use super::*;
+    use crate::test_util::kafka::{entries_with_random_data, new_namespace, EntryBuilder};
+
+    // Stores test context for a region.
+    struct RegionContext {
+        ns: NamespaceImpl,
+        entry_builder: EntryBuilder,
+        expected: Vec<EntryImpl>,
+    }
+
+    /// Prepares for a test in that a log store is constructed and a collection of topics is created.
+    async fn prepare(test_name: &str, num_topics: usize) -> (KafkaLogStore, Vec<Topic>) {
+        let broker_endpoints = get_broker_endpoints!(BROKER_ENDPOINTS_KEY);
+        let decorator = TopicDecorator::default()
+            .with_prefix(Affix::Fixed(test_name.to_string()))
+            .with_suffix(Affix::TimeNow);
```

Comment:
Why not just use the `uuid::new().to_string()`

Reply:
Either random number or current time would be ok. But I think time is more preferable since it's ordered.

Reply:
Do we rely on the order of topics?